exporters/prometheus/exporter.go

```diff
@@ -277,7 +277,11 @@ func addSumMetric[N int64 | float64](ch chan<- prometheus.Metric, sum metricdata
 			otel.Handle(err)
 			continue
 		}
-		m = addExemplars(m, dp.Exemplars)
+		// GaugeValues don't support Exemplars at this time
+		// https://github.com/prometheus/client_golang/blob/aef8aedb4b6e1fb8ac1c90790645169125594096/prometheus/metric.go#L199
+		if valueType == prometheus.CounterValue {
```

Comment:
Would it be clearer and more aligned with the comment to do the opposite condition, to emphasize that we only want to skip exemplars for gauges?

```suggestion
		if valueType != prometheus.GaugeValue {
```